scripts/micros_deploy.sh

```diff
@@ -5,7 +5,7 @@ rm -rf ../deploy/website
 mkdir ../deploy/website
 
 # Build
-cd .. && yarn build
+cd .. && bolt build
```

Comment:
Just calling out that this will break once my releasing packages change is merged.